client/webserver/site/src/js/app.ts

```diff
@@ -807,31 +807,20 @@ export default class Application {
    * haveActiveOrders returns whether or not there are active orders involving a
    * certain asset.
    */
-  haveAssetOrders (assetID: number): boolean {
+  haveActiveOrders (assetID: number): boolean {
```

Comment:
Renaming since looks like a typo (comment actually called `haveActiveOrders` which makes more sense to me).

Reply:
A mixed bag really. We've lost "asset" from the name, so less descriptive there, but we've added "active", so better on that part. I'm okay with the function signature providing some context for use, so this is fine, really. `haveActiveAssetOrders` would also be acceptable.